data/templates/run_end_to_end_tests.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Script to run end-to-end tests on Ubuntu Jenkins instance.
+#
+# This file is generated by l2tdevtools update-dependencies.py.
+
+# Fail on error.
+set -e
+
+CONFIGURATION=$$1;
+
+CONFIGURATION_FILE="$${CONFIGURATION}.ini";
+
+SOURCES_DIRECTORY="/media/greendale_images";
+REFERENCES_DIRECTORY="/media/greendale_images";
+
+RESULTS_DIRECTORY="plaso-out";
+
+./config/linux/gift_ppa_install.sh include-test;
+
+# Change path to test this script on Travis-CI.
+if test $${CONFIGURATION} = 'travis';
+then
+	SOURCES_DIRECTORY="test_data";
+	REFERENCES_DIRECTORY="test_data/end_to_end";
+	CONFIGURATION_FILE="config/jenkins/$${CONFIGURATION}.ini";
```

Comment:
Rather than $${CONFIGURATION}.ini, why not $CONFIGURATION_FILE?

Reply:
Unclear what you mean and why you want this changed.

Reply:
If you are asking why I changed it because it fails for multiple assignments to CONFIGURATION_FILE and I want to keep the assignments similar